README.md

```diff
@@ -69,6 +69,88 @@ We're now on the Maven Central Repository. Add the following to your POM file:
 
 The driver can be downloaded from the [Microsoft Download Center](https://www.microsoft.com/en-us/download/details.aspx?displaylang=en&id=11774)
 
+##Dependencies
+This project has following dependencies: 
+
+Compile Time:
+
+ - `azure-keyvault`
+
+Test Time:
+
+ - `junit:jar`
+
+If anybody wants to use driver as a run-time dependency but not using Azure Web Services for authentication then you either describe as optional dependency or exclude azure.
```

Comment:
Let's make this more generic in case we need to add more dependencies in the future.  I'd suggest something like this:

###Exclude Dependencies
If your wish to limit the number of run-time dependencies, and your project does not require the features named above, you can explicitly exclude them.  For example: